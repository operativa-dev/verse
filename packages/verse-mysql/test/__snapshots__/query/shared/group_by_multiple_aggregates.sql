-- Executing SQL: Parameters: []
select avg(`t1`.`AlbumId`) as `c0`, count(*) as `c1`
from `Album` as `t1`
group by `t1`.`ArtistId`

