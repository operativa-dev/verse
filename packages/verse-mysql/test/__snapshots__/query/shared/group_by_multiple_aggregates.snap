[
  [
    "2.5000",
    2,
  ],
  [
    "2.5000",
    2,
  ],
  [
    "5.0000",
    1,
  ],
  [
    "6.0000",
    1,
  ],
  [
    "7.0000",
    1,
  ],
  [
    "21.0000",
    2,
  ],
  [
    "9.0000",
    1,
  ],
  [
    "97.3333",
    3,
  ],
  [
    "12.0000",
    1,
  ],
  [
    "13.0000",
    1,
  ],
  [
    "14.5000",
    2,
  ],
  [
    "16.5000",
    2,
  ],
  [
    "18.0000",
    1,
  ],
  [
    "19.0000",
    1,
  ],
  [
    "20.0000",
    1,
  ],
  [
    "21.5000",
    2,
  ],
  [
    "23.0000",
    1,
  ],
  [
    "24.5000",
    2,
  ],
  [
    "26.5000",
    2,
  ],
  [
    "28.0000",
    1,
  ],
  [
    "39.7500",
    4,
  ],
  [
    "118.8571",
    14,
  ],
  [
    "31.0000",
    1,
  ],
  [
    "33.0000",
    1,
  ],
  [
    "86.0000",
    3,
  ],
  [
    "259.0000",
    1,
  ],
  [
    "47.0000",
    1,
  ],
  [
    "71.0000",
    1,
  ],
  [
    "158.5000",
    2,
  ],
  [
    "122.0000",
    1,
  ],
  [
    "140.3000",
    10,
  ],
  [
    "135.6667",
    3,
  ],
  [
    "81.5000",
    2,
  ],
  [
    "121.0000",
    2,
  ],
  [
    "64.0000",
    2,
  ],
  [
    "40.0000",
    1,
  ],
  [
    "41.0000",
    1,
  ],
  [
    "42.0000",
    1,
  ],
  [
    "59.1818",
    11,
  ],
  [
    "147.0000",
    3,
  ],
  [
    "84.6667",
    3,
  ],
  [
    "51.0000",
    1,
  ],
  [
    "52.0000",
    1,
  ],
  [
    "247.0000",
    1,
  ],
  [
    "54.5000",
    2,
  ],
  [
    "56.5000",
    2,
  ],
  [
    "67.0000",
    1,
  ],
  [
    "68.0000",
    1,
  ],
  [
    "69.5000",
    2,
  ],
  [
    "72.5000",
    2,
  ],
  [
    "75.5000",
    4,
  ],
  [
    "78.0000",
    1,
  ],
  [
    "80.5000",
    4,
  ],
  [
    "83.0000",
    1,
  ],
  [
    "84.0000",
    1,
  ],
  [
    "88.0000",
    1,
  ],
  [
    "91.0000",
    3,
  ],
  [
    "93.0000",
    1,
  ],
  [
    "104.0000",
    21,
  ],
  [
    "115.0000",
    1,
  ],
  [
    "117.0000",
    3,
  ],
  [
    "119.0000",
    1,
  ],
  [
    "120.0000",
    1,
  ],
  [
    "121.0000",
    1,
  ],
  [
    "123.0000",
    1,
  ],
  [
    "124.0000",
    1,
  ],
  [
    "125.0000",
    1,
  ],
  [
    "139.5000",
    2,
  ],
  [
    "141.0000",
    1,
  ],
  [
    "142.5000",
    2,
  ],
  [
    "144.0000",
    1,
  ],
  [
    "145.0000",
    1,
  ],
  [
    "146.0000",
    1,
  ],
  [
    "147.0000",
    1,
  ],
  [
    "160.0000",
    1,
  ],
  [
    "161.0000",
    1,
  ],
  [
    "162.0000",
    1,
  ],
  [
    "163.5000",
    2,
  ],
  [
    "165.0000",
    1,
  ],
  [
    "166.0000",
    1,
  ],
  [
    "168.0000",
    3,
  ],
  [
    "186.0000",
    6,
  ],
  [
    "175.0000",
    1,
  ],
  [
    "176.0000",
    1,
  ],
  [
    "177.0000",
    1,
  ],
  [
    "180.0000",
    5,
  ],
  [
    "183.0000",
    1,
  ],
  [
    "184.0000",
    1,
  ],
  [
    "187.0000",
    1,
  ],
  [
    "189.0000",
    3,
  ],
  [
    "191.0000",
    1,
  ],
  [
    "192.0000",
    1,
  ],
  [
    "194.0000",
    3,
  ],
  [
    "196.0000",
    1,
  ],
  [
    "199.5000",
    2,
  ],
  [
    "201.5000",
    2,
  ],
  [
    "203.0000",
    1,
  ],
  [
    "205.0000",
    1,
  ],
  [
    "206.0000",
    1,
  ],
  [
    "207.0000",
    1,
  ],
  [
    "208.0000",
    1,
  ],
  [
    "209.5000",
    2,
  ],
  [
    "211.0000",
    1,
  ],
  [
    "212.5000",
    2,
  ],
  [
    "214.0000",
    1,
  ],
  [
    "215.0000",
    1,
  ],
  [
    "217.0000",
    3,
  ],
  [
    "219.5000",
    2,
  ],
  [
    "221.0000",
    1,
  ],
  [
    "222.5000",
    2,
  ],
  [
    "224.5000",
    2,
  ],
  [
    "226.5000",
    2,
  ],
  [
    "228.0000",
    1,
  ],
  [
    "237.7500",
    4,
  ],
  [
    "237.9000",
    10,
  ],
  [
    "241.0000",
    1,
  ],
  [
    "243.5000",
    4,
  ],
  [
    "246.0000",
    1,
  ],
  [
    "248.0000",
    1,
  ],
  [
    "250.0000",
    3,
  ],
  [
    "252.0000",
    1,
  ],
  [
    "253.0000",
    1,
  ],
  [
    "254.0000",
    1,
  ],
  [
    "257.0000",
    1,
  ],
  [
    "258.0000",
    1,
  ],
  [
    "260.0000",
    1,
  ],
  [
    "262.0000",
    1,
  ],
  [
    "263.0000",
    1,
  ],
  [
    "264.0000",
    1,
  ],
  [
    "265.0000",
    1,
  ],
  [
    "266.0000",
    1,
  ],
  [
    "267.0000",
    1,
  ],
  [
    "268.0000",
    1,
  ],
  [
    "269.0000",
    1,
  ],
  [
    "270.0000",
    1,
  ],
  [
    "272.0000",
    1,
  ],
  [
    "273.0000",
    1,
  ],
  [
    "294.5000",
    2,
  ],
  [
    "275.0000",
    1,
  ],
  [
    "276.0000",
    1,
  ],
  [
    "277.0000",
    1,
  ],
  [
    "278.0000",
    1,
  ],
  [
    "279.0000",
    1,
  ],
  [
    "280.0000",
    1,
  ],
  [
    "281.0000",
    1,
  ],
  [
    "282.0000",
    1,
  ],
  [
    "283.0000",
    1,
  ],
  [
    "284.0000",
    1,
  ],
  [
    "285.0000",
    1,
  ],
  [
    "286.0000",
    1,
  ],
  [
    "287.0000",
    1,
  ],
  [
    "288.0000",
    1,
  ],
  [
    "289.0000",
    1,
  ],
  [
    "290.0000",
    1,
  ],
  [
    "291.0000",
    1,
  ],
  [
    "315.3333",
    3,
  ],
  [
    "293.0000",
    1,
  ],
  [
    "294.0000",
    1,
  ],
  [
    "295.0000",
    1,
  ],
  [
    "296.0000",
    1,
  ],
  [
    "297.0000",
    1,
  ],
  [
    "298.0000",
    1,
  ],
  [
    "299.0000",
    1,
  ],
  [
    "300.0000",
    1,
  ],
  [
    "301.0000",
    1,
  ],
  [
    "302.0000",
    1,
  ],
  [
    "303.0000",
    1,
  ],
  [
    "304.0000",
    1,
  ],
  [
    "305.0000",
    1,
  ],
  [
    "306.0000",
    1,
  ],
  [
    "307.0000",
    1,
  ],
  [
    "308.0000",
    1,
  ],
  [
    "309.0000",
    1,
  ],
  [
    "311.0000",
    2,
  ],
  [
    "313.0000",
    1,
  ],
  [
    "314.0000",
    1,
  ],
  [
    "324.0000",
    3,
  ],
  [
    "317.0000",
    1,
  ],
  [
    "318.0000",
    1,
  ],
  [
    "319.0000",
    1,
  ],
  [
    "321.5000",
    2,
  ],
  [
    "323.0000",
    1,
  ],
  [
    "324.0000",
    1,
  ],
  [
    "325.0000",
    1,
  ],
  [
    "326.0000",
    1,
  ],
  [
    "327.0000",
    1,
  ],
  [
    "328.0000",
    1,
  ],
  [
    "329.0000",
    1,
  ],
  [
    "330.0000",
    1,
  ],
  [
    "331.0000",
    1,
  ],
  [
    "332.0000",
    1,
  ],
  [
    "333.0000",
    1,
  ],
  [
    "334.0000",
    1,
  ],
  [
    "335.0000",
    1,
  ],
  [
    "337.0000",
    1,
  ],
  [
    "338.0000",
    1,
  ],
  [
    "339.0000",
    1,
  ],
  [
    "340.0000",
    1,
  ],
  [
    "341.0000",
    1,
  ],
  [
    "342.0000",
    1,
  ],
  [
    "344.0000",
    1,
  ],
  [
    "345.0000",
    1,
  ],
  [
    "346.0000",
    1,
  ],
  [
    "347.0000",
    1,
  ],
]